hub/apps/get-started/windows-developer-faq.yml

```
@@ -33,12 +34,12 @@ sections:
     questions:
       - question: What is the Windows App SDK?
         answer: |
-          The Windows App SDK is a Windows application development platform that allows you to create beautiful, modern, backwards-compatible (down to Win10 1809) desktop apps. WinUI 3 is the UI framework that ships with Windows App SDK.
+          The Windows App SDK is a Windows application development platform that allows you to create beautiful, modern, backwards-compatible (down to Windows 10 1809) desktop apps. WinUI 3 is the UI framework that ships with Windows App SDK.
```

Windows App SDK is not a platform.
https://github.com/microsoft/WindowsAppSDK/blob/main/docs/faq.md#is-the-windows-app-sdk-a-new-app-model

```suggestion
          The Windows App SDK contains a set of libraries that helps you build beautiful, modern, backwards-compatible (down to Windows 10 1809) desktop apps. WinUI 3 is the UI framework that ships with Windows App SDK.
```